update readme (#1)
DevExpress-Examples/how-to-save-a-document-in-the-richedit-control: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1401)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# How to: Save a Document in the RichEditControl

> This example illustrates how a document can be saved in the RichEditControl and customize this process.

Reviewer: ```suggestion
This example illustrates how to save a document in the RichEditControl and how to customize this process.
```

![application image](./media/image.png)

## Implementation Specifics

> The [RichEditControl.BeforeExport](https://docs.devexpress.com/WindowsForms/DevExpress.XtraRichEdit.RichEditControl.BeforeExport) event in handled to reduce the size of the RTF file before save. When you click the **Save Document to File** button, the warning message appears. If you click **Yes**, the [RtfDocumentExporterOptions](https://docs.devexpress.com/OfficeFileAPI/DevExpress.XtraRichEdit.Export.RtfDocumentExporterOptions) properties are specified to reduce the file size.

Reviewer: ```suggestion
The [RichEditControl.BeforeExport](https://docs.devexpress.com/WindowsForms/DevExpress.XtraRichEdit.RichEditControl.BeforeExport) event in handled to reduce the size of an RTF file before it is saved. When you click the **Save Document to File** button, a warning message appears. If you click **Yes**, the [RtfDocumentExporterOptions](https://docs.devexpress.com/OfficeFileAPI/DevExpress.XtraRichEdit.Export.RtfDocumentExporterOptions) properties are specified to reduce file size.
```

> A custom [SaveDocumentAsCommand](https://docs.devexpress.com/OfficeFileAPI/DevExpress.XtraRichEdit.Commands.SaveDocumentAsCommand) command is used to disable other formats but RTF in the dialog window.

Reviewer: You are missing a word or two here.
